Python/compile.c

```diff
@@ -171,6 +171,20 @@ struct instr {
     struct basicblock_ *i_except; /* target block when exception is raised */
 };
 
+/* One arg*/
+#define INSTR_SET_OP1(I, OP, ARG) \
+    do { \
+        struct instr *__instr__ptr__ = (I); \
+        assert(HAS_ARG(OP) || ((ARG) == 0)); \
+        (__instr__ptr__)->i_opcode = (OP); \
```

Comment:
No need for parentheses around `__instr__ptr__`, and use a single leading underscore. I think the C standard reserves double underscores for the compiler?
If you are worried about namespace pollution, then use an inline function.
(Which will make Victor happy :slightly_smiling_face:)